File: python/1046.py
i=input().split(" ")
s=int(i[0])
f=int(i[1])
if s == f:
    print("O JOGO DUROU 24 HORA(S)")
elif s > f:
    print("O JOGO DUROU "+str((24-s)+f)+" HORA(S)")
else:
    print("O JOGO DUROU "+str(f-s)+" HORA(S)")

File: C++/1046.cpp
#include <iostream>

using namespace std;

int main() {
    int s,f,c;
    cin >> s >> f;
    
    if(s==f){
        cout << "O JOGO DUROU 24 HORA(S)" << endl;
    }else if(s>f){
        c=(24-s)+f;
        cout << "O JOGO DUROU " << c << " HORA(S)" << endl;
    }else{
        c=f-s;
        cout << "O JOGO DUROU " << c << " HORA(S)" << endl;
    }
    return 0;
}